src/commands/Management/createMute.ts

```diff
@@ -26,6 +28,7 @@ export default class extends SkyraCommand {
 			await message.guild!.settings.update(GuildSettings.Roles.Muted, role, {
 				extraContext: { author: message.author.id }
 			});
+			await message.react(resolveEmoji(Emojis.GreenTick)!);
```

Comment:
Shouldn't this report the same thing as when you create?

Aka

```ts
await message.sendLocale('COMMAND_SUCCESS');
```

Reply:
I prefer reacting instead of stating the obvious (using the role "Muted"), but it's @kyranet's bot so I'll let him decide.